azurelinuxagent/ga/exthandlers.py

```diff
@@ -817,15 +853,20 @@ def handle_enable(self, ext_handler_i, extension):
                 uninstall_exit_code = ExtHandlersHandler._update_extension_handler_and_return_if_failed(
                     old_ext_handler_i, ext_handler_i, extension)
         else:
+            # Check extension policy and raise error if disallowed. Since the extension is not being re-downloaded, treat it
+            # as signed if its signature was previously validated on download.
+            extension_is_signed = os.path.exists(ext_handler_i.get_base_dir()) and signature_has_been_validated(ext_handler_i.get_base_dir())
```

Comment:
create helper method that returns the full path of the file, instead of using lib_dir evey time the file needs to be referenced